packages/flutter/lib/src/material/card.dart

```diff
@@ -65,51 +66,56 @@ import 'theme.dart';
 class Card extends StatelessWidget {
   /// Creates a material design card.
   ///
-  /// The [clipBehavior] and [elevation] arguments must not be null.
-  /// Additionally, the [elevation] must be non-negative.
+  /// The [elevation] must be non-negative.
   const Card({
     Key key,
     this.color,
-    this.elevation = 1.0,
+    this.elevation,
     this.shape,
-    this.margin = const EdgeInsets.all(4.0),
-    this.clipBehavior = Clip.none,
+    this.margin,
+    this.clipBehavior,
     this.child,
     this.semanticContainer = true,
-  }) : assert(elevation != null && elevation >= 0.0),
+  }) : assert(elevation == null || elevation >= 0.0),
        super(key: key);
 
   /// The card's background color.
   ///
   /// Defines the card's [Material.color].
   ///
-  /// The default color is defined by the ambient [Theme]: [ThemeData.cardColor].
+  /// If this property is null then [ThemeData.cardTheme.color] is used,
+  /// if that's null then [ThemeData.cardColor] is used.
   final Color color;
 
   /// The z-coordinate at which to place this card. This controls the size of
   /// the shadow below the card.
   ///
   /// Defines the card's [Material.elevation].
   ///
-  /// The default elevation is 1.0. The value is always non-negative.
+  /// If this property is null then [ThemeData.cardTheme.elevation] is used,
+  /// if that's null, the default value is 1.
```

Comment:
Although it doesn't really matter, we seem to still be writing 1.0 instead of 1

Reply:
done